doc/user-manual/getting-started/installation.rst

```diff
@@ -174,23 +174,34 @@ Nix or NixOS
 ------------
 
 Agda is part of the Nixpkgs collection that is used by
-https://nixos.org/nixos. To install Agda and agda-mode for Emacs,
-type:
+https://nixos.org/nixos. There are two ways to install Agda from nix:
 
-.. code-block:: bash
+* As Agda is a Haskell package available from hackage you can install
+  it like any other haskell package. To install Agda and agda-mode for
+  Emacs, type:
+
+  .. code-block:: bash
+
+    nix-env -f "<nixpkgs>" -iA haskellPackages.Agda
+
+* If you are tracking ``nixos-unstable`` or ``nixpkgs-unstable`` (the
+  default on MacOS) or are using a version of NixOS newer than 20.03
```

Comment:
This is only available from 20.09 onwards I believe

Reply:
updated this to make it clearer.